src/components/sbb-accordion/sbb-accordion.spec.ts

```diff
@@ -1,40 +1,42 @@
-import { SbbAccordion } from './sbb-accordion';
-import { newSpecPage } from '@stencil/core/testing';
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import './sbb-accordion';
```

Comment:
Is it on purpose that you don't import other related components?

Reply:
they are not strictly needed because they don't change the template but better import them all anyway, I'll do it 👍